FAQ.md

```diff
@@ -169,3 +170,36 @@ This one is very helpful if you want to know if a certain commit has landed in y
     ...
 
   ```
+## How can I install OKD without libvirt libraries?
```

Comment:
The plan is to have a dedicated libvirt installer in 4.8 as it really annoys people (sorry :( ).

Lets make it more generic - official installer container is what we use in CI, so this should be renamed to "Official installer container" and focus less on libvirt libs

Reply:
Okay

Reply:
I have changed that part. Please check again.